standard/standard_8.php

```diff
@@ -101,6 +102,7 @@ function header_register_callback(callable $callback): bool {}
  * @link https://secure.php.net/manual/en/function.getimagesizefromstring.php
  * @since 5.4
  */
+#[ArrayShape([0 => 'int', 1 => 'int', 2 => 'int', 3 => 'string', 'bits' => 'int', 'channels' => 'int', 'mime' => 'string'])]
```

Comment:
To me this looks more like:
```suggestion
#[ArrayShape(['width' => 'int', 'height' => 'int', 'itype' => 'int', 'temp' => 'string', 'bits' => 'int', 'channels' => 'int', 'mime' => 'string'])]
```
Comes from e.g. `add_index_long(return_value, 0, result->width);`: 
https://github.com/php/php-src/blob/cfa72ff3af144aa720aade9a311bf8160d57e8b9/ext/standard/image.c#L1507

Reply:
Simple run of the function gives
<img width="350" alt="Screenshot 2022-10-14 at 22 58 30" src="https://user-images.githubusercontent.com/37301285/195942984-0f32c01c-9752-48e4-b21e-c6ad79b9dd5b.png">